src/components/label-pill/LabelPill.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import Tooltip, { TooltipPosition } from '../tooltip';
+
+import './LabelPill.scss';
+
+export enum LabelPillStatus {
+    DEFAULT = 'default',
+    INFO = 'info',
+    HIGHLIGHT = 'highlight',
+    SUCCESS = 'success',
+    WARNING = 'warning',
+    ALERT = 'alert',
+    ERROR = 'error',
+}
+
+export interface LabelPillProps {
+    children: React.ReactNode;
+    tooltip?: string;
+    type?: LabelPillStatus;
+}
+
+const LabelPill = ({ children, type = LabelPillStatus.DEFAULT, tooltip = '', ...rest }: LabelPillProps) => {
+    const labelPillClasses = classNames(
+        'bdl-LabelPill',
+        type !== LabelPillStatus.DEFAULT ? `bdl-LabelPill--${type}` : '',
+    );
+    return (
+        <Tooltip position={TooltipPosition.TOP_CENTER} text={tooltip} isDisabled={!tooltip}>
```

Comment:
I feel like this will eventually expand to having all the props to customize the tooltip... maybe better left to the consumers to put their own tooltips on it?

Reply:
It really depends on the strength of the designs, IMO. if all of these had carefully considered layouts that aren't as flexible as `Tooltip`, I would prefer this method. If the positioning and such is just as flexible as the options provided to `Tooltip`, then they don't need to be combined. I don't know the answer, but this is a good point since things tend to be the latter. I'll decouple.